modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -50,7 +50,7 @@ $ oc patch argocd openshift-gitops \
 
 . In {rh-rhacm} 2.7 and later, the multicluster engine enables the `cluster-proxy-addon` feature by default.
 To disable this feature, apply the following patch to disable and remove the relevant hub cluster and managed cluster pods that are responsible for this add-on.
-
++
```

Comment:
I know you simply added a list continue, but while you are here, please consider updating line 52 to include some form of the of "run the following command".